frontend/src/components/MapView/index.tsx

```diff
@@ -596,15 +596,20 @@ function MapView({ classes }: MapViewProps) {
           >
             <Grid item>
               <Grid container spacing={1}>
-                {isShowingExtraFeatures && <GoToBoundaryDropdown />}
-                {appConfig.alertFormActive && isShowingExtraFeatures ? (
-                  <AlertForm
-                    isOpen={isAlertFormOpen}
-                    setOpen={setIsAlertFormOpen}
-                  />
-                ) : null}
+                {isShowingExtraFeatures && (
+                  <Grid item>
+                    <GoToBoundaryDropdown />
+                  </Grid>
+                )}
+                {appConfig.alertFormActive && isShowingExtraFeatures && (
```

Comment:
Since both are dependent on `isShowingExtraFeatures`, you could test for `isShowingExtraFeatures` only once at the top I think.